.github/workflows/test.yml

```diff
@@ -228,11 +237,18 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',   targets: ['x86_64-pc-windows-gnu'], rtools-version: '42'}
-          - {os: windows-latest, r: 'devel',   rust-version: 'stable-gnu',   targets: ['x86_64-pc-windows-gnu'], rtools-version: '42'}
-          # For R < 4.2, the MSVC toolchain is used to support cross-compilation for the 32-bit.
-          # TODO: Remove this runner when we drop the support for R < 4.2
-          - {os: windows-latest, r: '4.1',     rust-version: 'stable-msvc',  targets: ['x86_64-pc-windows-gnu', 'i686-pc-windows-gnu']}
+          # On Windows,
+          #
+          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
+          #   work. Since our main support is on MSVC, we mainly test MSVC here.
+          #   Also, at least one GNU case should be included so that we can
+          #   detect when something gets broken. 
+          # * for R < 4.2, the MSVC toolchain must be used to support
+          #   cross-compilation for the 32-bit. 
```

Comment:
```suggestion
          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
          #   work. Since we primarily support MSVC, we focus on testing MSVC here.
          #   Also, at least one GNU case should be included so that we can
          #   detect when something gets broken. 
          # * for R < 4.2, the MSVC toolchain must be used to support
          #   cross-compilation to 32-bit. 
```